contentcuration/contentcuration/frontend/channelEdit/components/NodeTreeNavigation.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+
+  <div>
+    <VBreadcrumbs
+      :items="breadcrumbsItems"
+      divider=">"
+    >
+      <template v-slot:item="props">
+        <span
```

Comment:
I checked that component and yes it would require updating its interface to make items able to emit an event on click. That's definitely a thing that would be useful to have implemented for our breadcrumbs in general. Though I checked "kolibri-design-system" and can see that there is [breadcrumbs component](https://github.com/learningequality/kolibri-design-system/blob/develop/lib/KBreadcrumbs.vue) so I suppose that it would make more sense to implement this interface rather there over time, makes sense?

@indirectlylit Is there a file where I could write down new use-cases for components so we can iterate on their interface?

Reply:
I just updated it on the [trash modal/move modal pr](https://github.com/learningequality/studio/pull/1804), so it should handle clicks soon. There will be a second pass much later to replace all components with Kolibri components, so it may make sense to consolidate it to the shared component to make it easier to update in one go anyways

Reply:
Thanks for the information. If I get to merge this before trash/move modals, I will do it as a follow-up. If your PR gets merged sooner, I'll rebase and use it from here.

Reply:
Follow-up issue here https://github.com/learningequality/studio/issues/1814